numpy/ctypeslib.py

```diff
@@ -527,8 +568,52 @@ def as_array(obj, shape=None):
 
 
     def as_ctypes(obj):
-        """Create and return a ctypes object from a numpy array.  Actually
-        anything that exposes the __array_interface__ is accepted."""
+        """
+            as_ctypes(obj)
+
+            Create and return a ctypes object from a numpy array.  Actually
+            anything that exposes the __array_interface__ is accepted.
```

Comment:
Maybe just briefly: from a NumPy array or any object implementing the ``__array_interface__`` protocol.